Kava/Kava/Source/PageObjects/Infrastructure/Elements/CollectionView.swift

```diff
@@ -65,7 +65,7 @@ open class CollectionView<TResultBlock : Block> : Element {
     }
     
     open func scrollForwardUntil<TCustomResultBlock : Block>(_ block: @escaping (() -> Bool), timeout: TimeInterval = 5, result: TCustomResultBlock.Type, constructingBlock builder: (() -> TCustomResultBlock)? = nil) -> TCustomResultBlock {
-        let predicate = NSPredicate { (_: Any, _: [String : Any]?) -> Bool in
```

Comment:
Note that the types for the parameters passed into an NSPredicate block in Swift 3 are `Any?, [String : Any]?`